libs/contrib/Data/Nat/Order/Properties.idr

```diff
@@ -38,31 +38,26 @@ notlteIsLT a b prf = notLTImpliesGTE
                                  $ lteReflection (S a) (S b)) prf'
   
 
-
-
 -- The converse to lteIsLTE:
 export
 LteIslte : (a, b : Nat) -> a `LTE` b -> a `lte` b = True
-                          -- DYI Syntax.PreorderReasoning from contrib
-LteIslte  a b a_lt_b with (the (x : Bool ** a `lte` b = x) (a `lte` b ** Refl))
- LteIslte a b a_lt_b | (True  ** prf) = prf
- LteIslte a b a_lt_b | (False ** prf) = void $ irreflexive {spo = Data.Nat.LT} a 
-   $ CalcWith {leq = LTE} $
-   |~ 1 + a
-   <~ 1 + b  ...(plusLteMonotoneLeft 1 a b a_lt_b)
-   <~ a      ...(Properties.notlteIsLT _ _ prf)
+LteIslte  a b a_lt_b = let u = reflect {c = True} (lteReflection a b) a_lt_b in ?hole
```

Comment:
You've left a `?hole` :D

Also: if you systematically need to pass `c` explicitly then it's probably
worth to make it explicit in the original proof.

Reply:
Sorry about that!

Reply:
It's a new pattern for me, so I had to play around with it first.